wrappers/aws/aws-sdk-go/aws/aws_test.go

```diff
@@ -180,6 +182,105 @@ var _ = Describe("epsagon aws sdk wrapper suite", func() {
 				Expect(res.Metadata["ConsumedCapacity"]).To(BeZero())
 			})
 		})
+		Context("sanity with simple dynamodb data - query items", func() {
+			var (
+				req          request.Request
+				world        string
+				tableName    string
+				param        dynamodb.QueryInput
+				data         dynamodb.QueryOutput
+				tracerConfig tracer.Config
+				items        []map[string]*dynamodb.AttributeValue
+			)
+			BeforeEach(func() {
+				world = "world"
+				tableName = "test-table"
+				param = dynamodb.QueryInput{
+					ExpressionAttributeNames: map[string]*string{
+						"hello": &world,
+					},
+					TableName: &tableName,
+				}
+				items = make([]map[string]*dynamodb.AttributeValue, 1)
+				items = append(items, map[string]*dynamodb.AttributeValue{
+					"hello": &dynamodb.AttributeValue{
+						S: &world,
+					},
+				})
+				data = dynamodb.QueryOutput{
+					Items: items,
+				}
+				req = request.Request{
+					ClientInfo: awsmetadata.ClientInfo{
+						ServiceName: "dynamodb",
+					},
+					Operation: &request.Operation{
+						Name: "Query",
+					},
+					Data:   &data,
+					Params: &param,
+				}
+				tracerConfig = tracer.Config{}
+				tracer.GlobalTracer = &tracer.MockedEpsagonTracer{
+					Config: &tracerConfig,
+				}
+			})
+			It("Extracts basic data", func() {
+				res := extractResourceInformation(&req, tracer.GlobalTracer)
+				Expect(res.Name).To(Equal(tableName))
+				Expect(res.Metadata).Should(HaveKey("Parameters"))
+				Expect(res.Metadata).Should(HaveKey("Response"))
+				var (
+					parameters map[string]interface{}
+					response   map[string]interface{}
+				)
+				parametersRaw := res.Metadata["Parameters"]
+				err := json.Unmarshal([]byte(parametersRaw), &parameters)
+				if err != nil {
+					Fail("json failed")
+				}
+				responseRaw := res.Metadata["Response"]
+				err = json.Unmarshal([]byte(responseRaw), &response)
+				if err != nil {
+					Fail("json failed")
```

Comment:
Could be nice to have a slightly different text, so we can easily identify which part failed (raw params or raw response)